node/libs/concurrency/src/sync/prunable_mpsc/tests.rs

```diff
@@ -0,0 +1,94 @@
+use crate::ctx;
+use assert_matches::assert_matches;
+use tokio::time::{timeout, Duration};
+
+// Test scenario:
+// 1. Pre-send two sets of 1000 values, so that the first set is expected to be pruned.
+// 2. Send a third set of 1000 values in parallel to receiving.
+#[tokio::test]
+async fn test_prunable_mpsc() {
+    let ctx = ctx::test_root(&ctx::RealClock);
+
+    #[derive(Debug, Clone)]
+    struct ValueType(usize, usize);
+
+    #[allow(clippy::type_complexity)]
+    let (send, mut recv): (
+        super::Sender<ValueType, Result<(), usize>>,
```

Comment:
nit, test modules are allowed to "use super::*;"

Reply:
Fixed.